src/Couchbase.Lite.Tests.Shared/DocumentTest.cs

```diff
@@ -2126,9 +2143,55 @@ public void TestMutableDocWithJsonString()
             var dic = PopulateDictData();
             var dicJson = JsonConvert.SerializeObject(dic);
             using (var md = new MutableDocument("doc1", dicJson)) {
-                var mdJson = md.ToJSON();
-                var mdJsonDict = JsonConvert.DeserializeObject<Dictionary<string, object>>(mdJson);
-                VerifyValuesInJson(dic, mdJsonDict);
+                foreach (var kvPair in dic) {
```

Comment:
Is this just copied and pasted from above?

Reply:
yeah... it's very very similar, but one is mutable dictionary and the other one is mutable document...

Reply:
Both of them implement `IMutableDictionary` so I think you can refactor into another method at a later point.